internal/instances/expander.go

```diff
@@ -419,7 +419,7 @@ func (e *Expander) GetResourceInstanceRepetitionData(addr addrs.AbsResourceInsta
 }
 
 // GetModuleCallInstanceKeys determines the child instance keys for one specific
```

Comment:
Thanks! I'll fix this one before merge just because it's not likely to cascade down onto other commits.

(For your other feedback, I'm going to keep it in mind for later work but leave it unchanged for now just to avoid churning this big pile of commits too much, since it's all experimental code we're explicitly intending to revisit anyway.)

Reply:
(For consistency I should really have named this method `GetResourceInstanceKeys`, but since changing that would impact the caller from another commit I'm just noting that in a comment here so I can refer back to it while planning later work.)